docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -433,6 +433,10 @@ dependencyManagement {
 ----
 ====
 
+=== DefaultOpenForTrafficCount and its effect on EurekaServer warmup time
+
+In order to enable the warmup time, Set `eureka.server.defaultOpenForTrafficCount=0`,It enables the warmup time, and the load testing app never gets its local cache emptied once the server comes back online.But expectedNumberOfRenewsThreshold will always be at 0 and never get updated for a standalone server.Also, Netflix Eureka's waitTimeInMsWhenSyncEmpty setting is not taken into account in Spring Cloud Eureka server at the beginning.
```

Comment:
Please capitalise "set" properly.

Reply:
Please remove the following sentences: "But expectedNumberOfRenewsThreshold will always be at 0 and never get updated for a standalone server." - it's not necessary.